idd/Energy+.idd.in

```diff
@@ -9235,9 +9235,216 @@ GeometryTransform,
+  A21, \field Space 20 Name
+       \type object-list
+       \object-list SpaceNames
+  A22, \field Space 21 Name
+       \type object-list
+       \object-list SpaceNames
+  A23, \field Space 22 Name
+       \type object-list
+       \object-list SpaceNames
+  A24, \field Space 23 Name
+       \type object-list
+       \object-list SpaceNames
+  A25, \field Space 24 Name
+       \type object-list
+       \object-list SpaceNames
+  A26, \field Space 25 Name
+       \type object-list
+       \object-list SpaceNames
+  A27, \field Space 26 Name
+       \type object-list
+       \object-list SpaceNames
+  A28, \field Space 27 Name
+       \type object-list
+       \object-list SpaceNames
+  A29, \field Space 28 Name
+       \type object-list
+       \object-list SpaceNames
+  A30, \field Space 29 Name
+       \type object-list
+       \object-list SpaceNames
+  A31, \field Space 30 Name
+       \type object-list
+       \object-list SpaceNames
+  A32, \field Space 31 Name
+       \type object-list
+       \object-list SpaceNames
+  A33, \field Space 32 Name
+       \type object-list
+       \object-list SpaceNames
+  A34, \field Space 33 Name
+       \type object-list
+       \object-list SpaceNames
+  A35, \field Space 34 Name
+       \type object-list
+       \object-list SpaceNames
+  A36, \field Space 35 Name
+       \type object-list
+       \object-list SpaceNames
+  A37, \field Space 36 Name
+       \type object-list
+       \object-list SpaceNames
+  A38, \field Space 37 Name
+       \type object-list
+       \object-list SpaceNames
+  A39, \field Space 38 Name
+       \type object-list
+       \object-list SpaceNames
+  A40, \field Space 39 Name
+       \type object-list
+       \object-list SpaceNames
+  A41, \field Space 40 Name
+       \type object-list
+       \object-list SpaceNames
+  A42, \field Space 41 Name
+       \type object-list
+       \object-list SpaceNames
+  A43, \field Space 42 Name
+       \type object-list
+       \object-list SpaceNames
+  A44, \field Space 43 Name
+       \type object-list
+       \object-list SpaceNames
+  A45, \field Space 44 Name
+       \type object-list
+       \object-list SpaceNames
+  A46, \field Space 45 Name
+       \type object-list
+       \object-list SpaceNames
+  A47, \field Space 46 Name
+       \type object-list
+       \object-list SpaceNames
+  A48, \field Space 47 Name
+       \type object-list
+       \object-list SpaceNames
+  A49, \field Space 48 Name
+       \type object-list
+       \object-list SpaceNames
+  A50, \field Space 49 Name
+       \type object-list
+       \object-list SpaceNames
+  A51; \field Space 50 Name
+       \type object-list
+       \object-list SpaceNames
+
 Zone,
-       \memo Defines a thermal zone of the building.
-  \format vertices
+       \memo Defines a thermal zone of the building. Every zone contains one or more Spaces.
+       \memo If no spaces are assigned to a zone, then a space named <Zone Name>-All will be generated..
```

Comment:
2 periods at end of this sentance.

Reply:
I'll make another pass on the idd to clean up.